.eslintignore

```diff
@@ -6,4 +6,5 @@
 *.svg
 *.sh
 *.md
-node_modules/**
\ No newline at end of file
+node_modules/**
+*.png
```

Comment:
This seems potentially overbearing.  What about `visual-regression-testing/**/*.png`?

Reply:
to be fair, we never want eslint to run on a png - this isn't the gitignore

Reply:
I'm actually confused why any of the file extensions are needed if we only are linting `.js, .jsx`?

Reply:
the whitelist doesn't seem to be working 🤷‍♀ try taking this out

Reply:
> Note: `--ext` is only used when the arguments are directories. If you use glob patterns or file names, then `--ext` is ignored.

I bet if we changed to `eslint src --ext .js,.jsx` this would work :/

Reply:
Did you try it?

Reply:
Yeah, but I'm having trouble pushing to this branch atm... I can follow-up afterward with a quick PR.

Reply:
@jbadan Good point. I didn't realize this was the eslint ignore file.  Carry on...